src/pybind/mgr/cephadm/serve.py

```diff
@@ -1693,6 +1680,46 @@ async def _get_container_image_info(self, image_name: str) -> ContainerInspectIn
         self.log.debug(f'image {image_name} -> {r}')
         return r
 
+    def _set_cephadm_command_timeout(self, timeout: Any) -> int:
+        def _safe_convert_to_int(t: Any) -> int:
+            if not isinstance(t, int):
```

Comment:
nitpick: calling `int()` on an int is not a problem, so you can elimintate a level of indent by dropping the isinstance check.